docs/sources/storage/_index.md

```diff
@@ -11,85 +11,110 @@ is then compressed and stored in chunks in object stores such as S3 or GCS, or
 even locally on the filesystem. A small index and highly compressed chunks
 simplifies the operation and significantly lowers the cost of Loki.
 
-Until Loki 2.0, index data was stored in a separate index.
+Prior to Loki 2.0, chunks and index data were stored in separate backends:
+object storage (or filesystem) for chunk data and NoSQL/Key-Value databases for index data.
 
-Loki 2.0 brings an index mechanism named 'boltdb-shipper' and is what we now call Single Store Loki.
-This index type only requires one store, the object store, for both the index and chunks.
+Loki 2.0 brings an index mechanism named 'boltdb-shipper' and is what we now call [Single Store](#loki-single-store).
```

Comment:
How does TSDB fit in here?  Should we update line 17 to point new users to TSDB instead of hoping they read all the way down to line 102?

Reply:
Was thinking exactly the same. I did not want to change the page for the 2.9 docs too much, but only add the deprecation notes. For the 3.0 docs, we should definitely re-organise the whole page.